lib/config/httpRoutes.js

```diff
@@ -217,7 +217,7 @@ const routes = [
   { verb: 'post', path: '/roles/_search', controller: 'security', action: 'searchRoles' },
   { verb: 'post', path: '/users/_search', controller: 'user', action: 'search', deprecated: { since: '2.10.3', message: 'Use this route instead: http://kuzzle:7512/user/_search' } },
   { verb: 'post', path: '/credentials/:strategy/:_id/_validate', controller: 'security', action: 'validateCredentials' },
-  { verb: 'post', path: '/_checkRights', controller: 'auth', action: 'checkRights' },
+  { verb: 'post', path: '/_checkRights', controller: 'user', action: 'checkRights' },
```

Comment:
I'm not sure about this change, to me we have 2 `checkRights` action because the first one is meant to be used by the current logged user to check his own rights and another one that allows admins to check a user rights.

So to me it's more the route l221 that need to be moved in the `user` controller

Reply:
Yep, that was an error. I never meant to change the `auth:checkRights`. I did update this accordingly tho, seems like you saw an outdated file.